### DISCUSSION TITLE
第2节课作业(23班)

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#171] [作业]
提交作业

**Comment by wangyijunlyy** [comment]
基础作业：
1.
<img width="1476" alt="image" src="https://github.com/InternLM/tutorial/assets/110682983/1116193f-c318-4688-8537-d1cc6c9f16e8">
2.
![image](https://github.com/InternLM/tutorial/assets/110682983/c893dfd4-e0d4-438d-a959-832f95e0636b)


进阶作业（选做）：
1.
<img width="1476" alt="image" src="https://github.com/InternLM/tutorial/assets/110682983/9fae4cf2-ac49-4924-9e23-7784bf6567c8">

2.
<img width="1476" alt="image" src="https://github.com/InternLM/tutorial/assets/110682983/eef7d97b-9279-446f-a9e5-7bccdb1a7bd1">

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection2-Easy-Fun-with-InternLM-Entertaining-Demo/

**Comment by yaoyaotaba** [comment]
https://zhuanlan.zhihu.com/p/677133266

**Comment by PlateAss** [comment]
https://plateass.github.io/work2

**Comment by yayayayaye** [comment]
https://b3x9893p70.feishu.cn/docx/PPTEdjoX7obhuxxvnN6c3pKdnKb?from=from_copylink

**Comment by Ace-may** [comment]
https://t5b1jxp2wd.feishu.cn/file/SmkebGeJ3ovfNAxVBFKcEhfKn3g?from=from_copylink

**Comment by relishLeisure** [comment]
http://t.csdnimg.cn/OiBye

**Comment by 1131764933** [comment]
https://github.com/InternLM/tutorial/discussions/171

**Comment by 1131764933** [comment]
https://github.com/InternLM/tutorial/discussions/171
![大模型编写300字故事](https://github.com/InternLM/tutorial/assets/80241043/31cba83c-afe2-44e5-bd8f-68315967974a)
![下载config json](https://github.com/InternLM/tutorial/assets/80241043/cf94173a-4c4c-4f8f-89ce-ebb550274689)

**Comment by HuJijin** [comment]
https://polished-collision-b6f.notion.site/c514f46379774695818d77dd1eb26f49?pvs=4

**Comment by YangZhC** [comment]
https://blog.csdn.net/donotpython/article/details/135761516

**Comment by 1131764933** [comment]
![Uploading 构建知识库.png…]()

**Comment by 1131764933** [comment]
![第五节](https://github.com/InternLM/tutorial/assets/80241043/bf4de110-5959-4d00-a84b-e08d0bd37f6f)
![第五节课](https://github.com/InternLM/tutorial/assets/80241043/5aa6ce2c-f5dd-4d0a-8430-0c795ffeb424)

**Comment by 1131764933** [comment]
![第四节](https://github.com/InternLM/tutorial/assets/80241043/1952e166-afbf-4173-ac59-f6041414dfe7)

**Comment by 1131764933** [comment]
![第六节](https://github.com/InternLM/tutorial/assets/80241043/877421a1-2303-4cbb-884e-3a1fecbca8a3)
![image](https://github.com/InternLM/tutorial/assets/80241043/2b1ad45f-cf9e-4c0d-a77a-4c863d838cb7)
![image](https://github.com/InternLM/tutorial/assets/80241043/171164c9-0e34-43c7-977c-babfb30979b1)
